### PR TITLE
Docs/fixes and minor edits

### PR DESCRIPTION
**Pull request by PipKat**
This PR cleans up the existing *PyAnsys developer's doc* so that it follows our style guidelines. Until I am done editing all of the files, I plan on keeping it a draft PR. @RobPasMue If you want me to make it ready for review sooner (perhaps preferring more frequent, smaller PRs to ease the review process), let me know! Note that I have the Vale error again indicating that there are too many results (annotations) in the diff.

Changes made so far:

- Set up code block examples using tabs as in others other parts of the dev guide. (I'm not sure if the "Define a caption and name for referencing a code block" section is showing correctly what would render in the documentation.)
- Replaced occurrences of "via" as the Google style guide and others recommend avoiding this word.
- Removed glossary as its inclusion (and correctness) was questioned long ago.
- Add content on the naming format required for Python folders and files (underscore must be used as space delimiter) and recommended for RST folder and files (hyphen should be used for search optimization of the HTML documentation).
- Add information on how to comment out lines if you are not using the ``sphinx.ext.todo`` extension but why adding and using this native Sphinx extension is recommended.
- Document how to use the ``git clean -fdx .`` command to remove untracked files and directories.
- Add info about how new user overall ``index.rst`` files (doc landing pages) use a grid of cards to present the organization of the documentation in a visual manner.

### CODE REVIEW

**Comment by RobPasMue** [comment]
@dipinknair - since you've been working on vale recently, could you help out @PipKat on resolving the issues?

**Comment by PipKat** [comment]
@RobPasMue @dipinknair Hi, Roberto. Dipin and I met on Friday afternoon to look at the Vale issues. We both agreed that it would be best for you to review and force merge this PR and then create and assign an issue for him to investigate.  We don't want to ignore the Google HEADING rule to use sentence case. However, it's through errors for titles that are using sentence case. (The only capital letter in the title is the one on the initial word.) It's also throwing errors when proper nouns are used in titles, but those proper nouns should be fine because the exist in the project's ``accept.txt`` file. It will be easier to Dipin to investigate after this very large PR is merged. I have finished reviewing content in these dev guide sections: "Getting started," "How-to," and "Style." I can finish the last two remaining sections after Dipin's investigation. (Plus, I have some Dev portal and Ansys Store work to do early next week.)

**Comment by RobPasMue** [comment]
Sounds good @dipinknair @PipKat - I will do a final review on Monday morning. Have a nice weekend!

File: doc/source/content-writing/content-how-tos/create-PR.rst [hunk 1]
```diff
@@ -459,3 +459,33 @@ One the PR is merged, use your GitHub tool to pull all changes from the remote m
...
+
+To remove untracked files and directories from your working directory, from the
+``doc`` folder, periodically run this command:
+
+``git clean -fdx .``
+
+For more information on this Git command, see :ref:`git_clean`.
+
+When you next run ``pre-commit``, the code style tools configured for
+your PyAnsy project must be initialized once again. For more information,
```
Comment (RobPasMue) [inline doc/source/content-writing/content-how-tos/create-PR.rst:474]:
```suggestion
your PyAnsys project must be initialized once again. For more information,
```

**Review by RobPasMue — APPROVED** [review]
LGTM! Merging!

**Pull request merged**